### PR TITLE
Feature/vr 3688 commit in multiple locations fix

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/src/main/resources/liquibase/create-versioning-tables-changelog2.xml [hunk 1]
```diff
@@ -296,4 +296,17 @@
 				constraintName="ck_versioning_modeldb_entity_mapping"
 				tableName="versioning_modeldb_entity_mapping"/>
 	</changeSet>
+	<changeSet id="changeKeysOfFolderElement" author="lezhevg">
+		<dropTable tableName="folder_element"/>
```
Comment (conradoverta) [inline backend/src/main/resources/liquibase/create-versioning-tables-changelog2.xml:317]: I’m not sure I follow why we need to drop the table

**Review by conradoverta — CHANGES_REQUESTED** [review]

File: backend/src/main/resources/liquibase/create-versioning-tables-changelog2.xml (hunk 1 above)
Comment (evgeniilezhnin) [reply backend/src/main/resources/liquibase/create-versioning-tables-changelog2.xml:317]: fixed

**Comment by ravishetye** [comment]
@evgeniilezhnin this needs to be edited as per the recommendation in slack discussion, right?

**Comment by evgeniilezhnin** [comment]
> @evgeniilezhnin this needs to be edited as per the recommendation in slack discussion, right?

had already did that

File: backend/src/main/resources/liquibase/create-versioning-tables-changelog2.xml (hunk 1 above)
Comment (ravishetye) [reply backend/src/main/resources/liquibase/create-versioning-tables-changelog2.xml:317]: break them into changeset as mentioned in the message. A single changeset if interrupted won't rollback create table.

**Comment by ravishetye** [comment]
> > @evgeniilezhnin this needs to be edited as per the recommendation in slack discussion, right?
> 
> had already did that

break them into changeset as mentioned in the message. A single changeset if interrupted won't rollback create table.

**Comment by evgeniilezhnin** [comment]
> > > @evgeniilezhnin this needs to be edited as per the recommendation in slack discussion, right?
> > 
> > 
> > had already did that
> 
> break them into changeset as mentioned in the message. A single changeset if interrupted won't rollback create table.

done

**Review by ravishetye — APPROVED** [review]

**Pull request merged**